src/overlays/actors/ovl_Obj_Grass/z_obj_grass.h

```diff
@@ -7,40 +7,43 @@
 struct ObjGrass;
 struct ObjGrassCarry;
 
-typedef struct ObjGrassStruct1_1 {
-    /* 0x00 */ Vec3f unk_00;
-    /* 0x0C */ s16 unk_0C;
-    /* 0x0E */ s8 unk_0E;
-    /* 0x0F */ u8 unk_0F;
-    /* 0x10 */ u8 unk_10;
-} ObjGrassStruct1_1; // size = 0x14
+#define OBJ_GRASS_GROUP_DRAW 1
+#define OBJ_GRASS_NUM_COLLIDERS 20
+
+typedef struct ObjGrassElement {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ s16 rotY;
+    /* 0x0E */ s8 dropTable;
+    /* 0x0F */ u8 elem_flags;
```

Comment:
```suggestion
    /* 0x0F */ u8 elemFlags;
```
btw could you make defines for them?